tasks.rb

```diff
@@ -72,14 +75,29 @@ def create_hpxmls
       end
 
       # Re-generate stochastic schedule CSV?
-      csv_path = json_input['schedules_filepaths'].to_s.split(',').map(&:strip).find { |fp| fp.include? 'occupancy-stochastic' }
-      if (not csv_path.nil?) && (not schedules_regenerated.include? csv_path)
+      prev_csv_path = nil
+      csv_path = json_input['schedules_filepaths'].to_s.split(',').map(&:strip).find { |fp| fp.include? stochastic_sched_basename }
+      if hpxml_path.include?('base-schedules-detailed-occupancy-stochastic-10-mins.xml') ||
+         hpxml_path.include?('base-simcontrol-timestep-10-mins-occupancy-stochastic-60-mins.xml')
+        # These sample files test simulation timesteps that differ from the stochastic schedule timestep. If we
+        # call the BuildResidentialScheduleFile measure below, it will generate a stochastic schedule that matches
+        # the simulation timestep; so we'll avoid the call and just use the stochastic schedule generated from
+        # another HPXML file.
+        csv_path = nil
+      end
+      if not csv_path.nil?
```

Comment:
I think it might be better to define this list upfront.

```suggestion
      schedule_skip_list = [
          'base-schedules-detailed-occupancy-stochastic-10-mins.xml',
          'base-simcontrol-timestep-10-mins-occupancy-stochastic-60-mins.xml'
      ]  # This definition can be moved up along with the comment explaining which xml files should be here
      if csv_path and !schedule_skip_list.include?(File.basename(hpxml_path))
```

Reply:
Good suggestion, will do.